src/features/venue/components/OpeningHoursStatus/getOpeningHoursStatus.native.test.ts

```diff
@@ -371,60 +420,94 @@ describe('OpeningHoursStatusViewModel', () => {
         openingHours: {
           MONDAY: [{ open: '09:00', close: '19:00' }],
         },
-        currentDate: dayFactory.monday('18:00:00'),
-        expectedNextChange: dayFactory.monday('19:00:00'),
+        currentDate: paris.monday('18:00:00'),
+        expectedNextChange: paris.monday('19:00:00'),
       },
       {
         openingHours: {
           MONDAY: [{ open: '09:00', close: '19:00' }],
         },
-        currentDate: dayFactory.monday('07:00:00'),
-        expectedNextChange: dayFactory.monday('09:00:00'),
+        currentDate: paris.monday('07:00:00'),
+        expectedNextChange: paris.monday('09:00:00'),
       },
       {
         openingHours: {
           MONDAY: [{ open: '09:00', close: '19:00' }],
         },
-        currentDate: dayFactory.monday('12:00:00'),
-        expectedNextChange: dayFactory.monday('19:00:00'),
+        currentDate: paris.monday('12:00:00'),
+        expectedNextChange: paris.monday('19:00:00'),
       },
       {
         openingHours: {
           MONDAY: [{ open: '09:00', close: '19:00' }],
         },
-        currentDate: dayFactory.monday('08:30:00'),
-        expectedNextChange: dayFactory.monday('09:00:00'),
+        currentDate: paris.monday('08:30:00'),
+        expectedNextChange: paris.monday('09:00:00'),
       },
       {
         openingHours: {
           MONDAY: [{ open: '09:00', close: '19:00' }],
         },
-        currentDate: dayFactory.monday('19:30:00'),
-        expectedNextChange: addWeeks(dayFactory.monday('09:00:00'), 1),
+        currentDate: paris.monday('19:30:00'),
+        expectedNextChange: addWeeks(paris.monday('09:00:00'), 1),
       },
       {
         openingHours: {
           MONDAY: [],
           FRIDAY: undefined,
         },
-        currentDate: dayFactory.monday('19:00:00'),
+        currentDate: paris.monday('19:00:00'),
         expectedNextChange: undefined,
       },
       {
         openingHours: {
           MONDAY: [{ open: '09:00', close: '18:00' }],
           TUESDAY: [{ open: '09:00', close: '18:00' }],
         },
-        currentDate: dayFactory.monday('19:00:00'),
-        expectedNextChange: dayFactory.tuesday('09:00:00'),
+        currentDate: paris.monday('19:00:00'),
+        expectedNextChange: paris.tuesday('09:00:00'),
       },
     ])(
       'should indicate next state change date $expectedNextChange',
       ({ openingHours, currentDate, expectedNextChange }) => {
-        const viewModel = getOpeningHoursStatus({ openingHours, currentDate })
+        const viewModel = getOpeningHoursStatus({
+          openingHours,
+          currentDate,
+          timezone: PARIS_TIMEZONE,
+        })
 
         expect(viewModel.nextChangeTime).toEqual(expectedNextChange)
       }
     )
   })
+
+  describe('Venue is not on same timezone', () => {
+    test('user is at paris and venue is in guadeloupe', () => {
```

Comment:
dommage juste après tu mets ça correctement! C'est "in" paris, pas "at" paris :)